web_widget_color/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Odoo, Open Source Web Widget Color
+# Copyright (C) 2012 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# Copyright (C) 2014 Anybox <http://anybox.fr>
+# Copyright (C) 2015 Taktik SA <http://taktik.be>
+# Copyright (C) 2018 Alexandre Díaz <dev@redneboa.es>
+#
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).#
+{
+    'name': "Web Widget Color",
+    'category': "web",
+    'version': "12.0.1.0.0",
+    "author": "Savoir-faire Linux, "
```

Comment:
IMO add `website` key, and URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)